components/panel/style.scss

```diff
@@ -74,6 +74,7 @@
 	font-weight: 600;
 	text-align: left;
 	color: $dark-gray-900;
+	background-color: $white;
```

Comment:
Can we apply a `background: none` to the default `.components-button` instead? It seems reasonable to reset the background for all Button components

Reply:
Hey @youknowriad, thanks for the review. It definitely makes more sense that way, just pushed the fix.